src/components/inputModal/index.tsx

```diff
@@ -1,62 +1,93 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import { Modal, Form } from "react-bootstrap";
-import React, { useEffect } from "react";
+import React, { useEffect, useMemo } from "react";
 import { useForm, Controller } from "react-hook-form";
 import { yupResolver } from "@hookform/resolvers/yup";
 import FormButtonGroup from "components/inputModal/FormAction";
 import { schema } from "utils/inputFormSchema";
 import { MODAL_TYPE } from "utils/enums";
-import InputTextField from "components/inputModal/InputTextField";
-import InputSelectField from "components/inputModal/InputSelect";
-import {grades, subjects} from "utils/index";
+import InputTextField from "components/inputModal/inputTextField";
+import InputSelectField from "components/inputModal/inputSelect";
+import { grades, subjects } from "utils/index";
+import { IStudentRaw } from "state/ducks/students/types";
+import { ActionType } from "typesafe-actions";
+import { addStudent, updateStudent } from "state/ducks/students/actions";
```

Comment:
it shouldn't be `state`, it should be `store`

Reply:
the reference link you provided initially used **state** so i followed that pattern, let me know if you still want me to change it

Reply:
pls change it